drivers/hmis/app/graphql/schema.graphql

```diff
@@ -3981,6 +3981,32 @@ type FormDefinitionsPaginated {
   pagesCount: Int!
 }
 
+"""
+FormIdentifier
+"""
+type FormIdentifier {
+  allVersions(limit: Int, offset: Int): FormDefinitionsPaginated!
+  displayVersion: FormDefinition!
+  draft: FormDefinition
+  id: String!
+  identifier: String!
+  published: FormDefinition
```

Comment:
It seems inconsistent that `displayVersion` and `allVersion` have "version" in the name and they resolve FormDefinitions, whereas `draft` and `published` _don't_ have `Version` in the name, even though they resolve the same thing. I would have thought `draft`/`published` were booleans based on the name.

 I suggest standardizing, to all use `_versions` suffix (or `_form`, but maybe we have enough "form" language 😅 )